feature/karmalist/karmalistfeature.go

```diff
@@ -0,0 +1,50 @@
+package karmalist
+
+import (
+	"github.com/jakevoytko/crbot/api"
+	"github.com/jakevoytko/crbot/feature"
+	stringmap "github.com/jakevoytko/go-stringmap"
+)
+
+// Feature is a Feature that lists commands that are available.
+type Feature struct {
+	featureRegistry *feature.Registry
+	karmaMap      stringmap.StringMap
+	gist            api.Gist
```

Comment:
If you run `go fmt ./...` from the root directory, does it produce any changes?

Reply:
Yup, it sure does. I'll remember to run that to keep things pretty.